lib/network/modules/components/physics/FA2BasedCentralGravitySolver.js

```diff
@@ -1,15 +1,14 @@
 import CentralGravitySolver from "./CentralGravitySolver"
 
 /**
- * @class ForceAtlas2BasedCentralGravitySolver
+ * @param {Object} body
+ * @param {{physicsNodeIndices: Array, physicsEdgeIndices: Array, forces: {}, velocities: {}}} physicsBody
```

Comment:
jsdoc can deal with this?? Or is this for the typescript conversion?

This (and related) looks like a good candidate for `@typedef`.